### ISSUE TITLE
Problem: Not enough issue labels

### ISSUE THREAD

**Issue by MakaryGo** [magicstone-dev/ecko#323]
*We could use some more issue labels in this repository*
I believe that if we added more general labels, ie. `documentation` , `UX` etc.  it would help us organise work on them.

**Comment by weex** [comment]
So per [C4](https://rfc.zeromq.org/spec/42/#24-development-process), I would rephrase this as a problem something like: *It's hard to find specific kinds of issues.* or *It's hard to group issues by type* but I would then ask why are those worth the overhead of adding and maintaining various labels?

My own experience is that search works pretty well and since we only have a couple pages of issues, it's not a huge problem to scroll though them, but I could definitely see it growing over time.

**Comment by MakaryGo** [comment]
Thank you very much for explaining, I'm still learning the C4 :)
My reasoning just FYI:
I'd love to contribute, but because of my rather limited skillset, I'm able to help almost only with documentation related issues, so I thought that having label for them would be nice. I fully accept your point though, for now there is no absolute need for it.
Closing.
